File: WEEK7/Js/bj10_11758.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const p = [];
for (let i = 0; i < 3; i++) {
    p.push(input[i].split(' ').map(Number));
}

const result = (p[1][0] - p[0][0]) * (p[2][1] - p[0][1]) - (p[2][0] - p[0][0]) * (p[1][1] - p[0][1]);

if (result > 0) console.log(1);
else if (result < 0) console.log(-1);
else console.log(0);

File: WEEK7/Python/bj10_11758.py
import sys

p = []
for _ in range(3):
    p.append(list(map(int, sys.stdin.readline().split())))

result = (p[1][0] - p[0][0]) * (p[2][1] - p[0][1]) - (p[2][0] - p[0][0]) * (
    p[1][1] - p[0][1]
)

if result > 0:
    print(1)
elif result < 0:
    print(-1)
else:
    print(0)
